### PR TITLE
8222825: ARM32 SIGILL issue on single core CPU (not supported PLDW instruction)

### PR DESCRIPTION
**Pull request by bulasevich**
This is the request to backport the fix to jdk11u.
The change fixes SIGILL on ARM machines without the Multiprocessing Extensions and brings no risk for other platforms.

The original patch [1] has minor conflicts with os::is_MP [2] and AArch64 Port [3] removal updates.
 [1] https://github.com/openjdk/jdk/commit/989fa190
[2] https://github.com/openjdk/jdk/commit/a3cd6a1a
[3] https://github.com/openjdk/jdk/commit/05027c12

The jtreg tests are Ok.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8222825](https://bugs.openjdk.java.net/browse/JDK-8222825): ARM32 SIGILL issue on single core CPU (not supported PLDW instruction)

### Reviewers
 * [Aleksey Shipilev](https://openjdk.java.net/census#shade) (@shipilev - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk11u-dev pull/652/head:pull/652` \
`$ git checkout pull/652`

Update a local copy of the PR: \
`$ git checkout pull/652` \
`$ git pull https://git.openjdk.java.net/jdk11u-dev pull/652/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 652`

View PR using the GUI difftool: \
`$ git pr show -t 652`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk11u-dev/pull/652.diff">https://git.openjdk.java.net/jdk11u-dev/pull/652.diff</a>

</details>

### CODE REVIEW

**Comment by snazarkin** [comment]
LGTM but I'm not formal reviewer. Checked with build and run on RPI4

**Review by shipilev — APPROVED** [review]
Looks fine to me. 

The original patch has this definition, which is missing from the backport, but I don't see how it affects anything.

```
+  static bool _has_mp_ext;
```

(Old) AArch64 code looks fine too (not that we care at this junction), we should be using the same match rule.

**Comment by RealCLanger** [comment]
Hm, this issue did not have maintainer approval AFAICS...

**Comment by shipilev** [comment]
> Hm, this issue did not have maintainer approval AFAICS...

Oh, my bad. Let me file one retroactively.

**Pull request closed without merging**
